bot/cogs/embeds.py

```diff
@@ -297,10 +297,13 @@ async def has_active_embed(self, ctx):
         if ctx.author in self.embeds:
             return True
         else:
-            await ctx.send(f":x: {ctx.author.mention} No active embed found, are you in embed building mode? (`{Bot.prefix}help Embeds`)")
+            await ctx.send(
+                f"{Emojis.cross_mark} {ctx.author.mention} No active embed found, "
+                f"are you in embed building mode? (`{Bot.prefix}help Embeds`)"
```

Comment:
```suggestion
                f"are you in embed building mode? (`{BotConstant.prefix}help Embeds`)"
```
`Bot` is a reference to `bot.bot.Bot` which has no attribute `prefix`
This should be  `bot.constants.Bot`